### ISSUE TITLE
Can you give the codes about training？

### ISSUE THREAD

**Issue by bityangke** [escorciav/daps#1]
First of all， Thank you very much for sharing the codes.
Could you please share the training codes? I just want to know something about how to write the training codes but do not care whether the codes pass functional test. 
At least the codes of loss function in the paper?
Thanks anyways!

**Comment by escorciav** [comment]
I released code for training DAPs model. You may be interested in [this part](https://github.com/escorciav/deep-action-proposals/blob/master/daps/learning.py#L115-L128)

**Comment by bityangke** [comment]
Thank you very much！
